File: JAVA/aksesoris.java
class aksesoris extends petshop{
    private String jenis,bahan,warna;

    public aksesoris(){
        this.jenis="";this.bahan="";this.warna="";
    }
    
    public aksesoris(String jenis,String bahan,String warna){
        this.jenis=jenis;
        this.bahan=bahan;
        this.warna=warna;
    }

    public String getjenis(){
        return this.jenis;
    }
    public void setjenis(String jenis){
        this.jenis = jenis;
    }
    
    public String getbahan(){
        return this.bahan;
    }
    public void setbahan(String bahan){
        this.bahan = bahan;
    }
    
    public String getwarna(){
        return this.warna;
    }
    public void setwarna(String warna){
        this.warna = warna;
    }
    
}

File: PHP/aksesoris.php
<?php
require('petshop.php');
class aksesoris extends petshop{
    private $jenis='';
    private $bahan='';
    private $warna='';
    public function __construct($jenis = '',$bahan = '',$warna = ''){
        $this->jenis= $jenis;
        $this->bahan= $bahan;
        $this->warna= $warna;
    }
    
    public function setjenis($jenis){
        $this->jenis= $jenis;
    }
    public function getjenis(){
        return $this->jenis;
    }
    
    public function setbahan($bahan){
        $this->bahan= $bahan;
    }
    public function getbahan(){
        return $this->bahan;
    }
    
    public function setwarna($warna){
        $this->warna= $warna;
    }
    public function getwarna(){
        return $this->warna;
    }
}